### ISSUE TITLE
[Feature Request] Archive Vault / Wallet *via* API

### ISSUE THREAD

**Issue by cyrilchapon** [fireblocks/fireblocks-sdk-js#169]
In a context where we abstract the wallet creation behind a REST API of our own; we have a use case where we probably should archive the vault account + wallet linked to a user.

I discovered it can be done via the console; but not via API.

Any plans to support it ?

**Comment by tomerrrrrr** [comment]
Hi @cyrilchapon ! Hiding the vaults from the UI is available via the hiddenOnUI property. https://developers.fireblocks.com/reference/post_vault-accounts-vaultaccountid-hide

**Comment by cyrilchapon** [comment]
From my tests, the "hidden" state is different from the archived one

**Comment by tomerrrrrr** [comment]
that is correct - we have 2 different things .. hiding , and archiving. You can see the comparison table of these two here - https://support.fireblocks.io/hc/en-us - and basically, if you plan on having more than 50 vault accounts, hiding is probably the better approach, which is why we focused on the hiding experience in our APIs. I would love to hear if you have more questions or benefits for the archiving approach over the hiding approach, that may work better with our APIs, and we can revisit this.

**Comment by cyrilchapon** [comment]
@tomerrrrrr ok thanks.

> I would love to hear if you have more questions or benefits for the archiving approach over the hiding approach, that may work better with our APIs, and we can revisit this.

Well they're basically 2 separates use-case IMHO. The "hiding" approach is essentially here not-to-mess-the-ui with many-wallets applications (and I think we will need this eventually); while the "archiving" approach is sort of a "soft-delete" mecanism.

In other words :
- "Hiding" = "I'm using it, but it's not as crucial to show in the UI"
- "Archiving" = "I'm not using, and I don't want to see it until I really need for special purposes"
